en/views/helpers/form.rst

```diff
@@ -16,39 +16,53 @@ Starting a Form
 
 .. php:method:: create(mixed $model = null, array $options = [])
 
+* ``$model`` - The context for which the form is being defined. Can be an ORM
+  entity, ORM resultset, array of metadata or ``false/null`` (to make a
+  model-less form).
+* ``$options`` - An array of options and/or HTML attributes.
+
 The first method you'll need to use in order to take advantage of the FormHelper
 is ``create()``. This method outputs an opening form tag.
 
 All parameters are optional. If ``create()`` is called with no parameters
 supplied, it assumes you are building a form that submits to the current
 controller, via the current URL. The default method for form submission is POST.
-If you were to call ``create()`` inside the view for UsersController::add(), you
-would see something like the following output in the rendered view:
+If you were to call ``create()`` inside the view for ``UsersController::add()``,
+you would see something like the following output in the rendered view:
 
 .. code-block:: html
 
     <form method="post" action="/users/add">
 
 The ``$model`` argument is used as the form's 'context'. There are several
-built-in form contexts and you can add your own, which we'll cover in the next
-section. The built-in providers map to the following values of ``$model``:
-
-* An ``Entity`` instance or, an iterator map to the ``EntityContext``, this
-  context allows FormHelper to work with results from the built-in ORM.
-* An array containing the ``schema`` key, maps to ``ArrayContext`` which allows
-  you to create simple data structures to build forms against.
-* ``null`` and ``false`` map to the ``NullContext``, this context class simply
-  satisifies the interface FormHelper requires. This context is useful if you
-  want to build a short form that doesn't require ORM persistence.
-
-All contexts classes also have access to the request data, making it simpler to
+built-in form contexts and you can add your own, which we'll cover below, in
+a following section. The built-in providers map to the following values of
+``$model``:
+
+* An ``Entity`` instance or an iterator will map to
+  `EntityContext <https://api.cakephp.org/3.3/class-Cake.View.Form.EntityContext.html>`_;
```

Comment:
We already have, since quite a while, on this page, the link to the API, inside the List Of Templates section. To be frank I found and still find it highly valuable, as I recently used it quite often. It's true it still points to the 3.2 version of the documentation but that part didn't seem to change in 3.3.

The thing is, that the enumeration list in the beginning of the page, as it is now, is of too little use to any first-comer without any pointer towards those classes; you won't know neither where these classes are. In fact it's not obvious at first view even that they are classes for that matter; when I first arrived on the page I couldn't make heads or tails of this list and figured out that it's referring internal classes, from the small hint in the text of the 3rd option which is *null*, and then it took me a bit of search to actually find them.
The list tells us even less what kind of options/arguments we could use to customize them.

The whole point of the links is to help other users avoid this hurdle and land on the corresponding API page then figure it out, themselves, from there.

I understand the argument about the API code maybe being changed in the future and the link becoming a bit stale; but I would argue that a user savvy enough to start creating custom contexts, will know to search for the last version of the API, starting from the corresponding page where s/he lands.
On the other hand if the API changes that much that these classes are completely removed or change names this section of the page will be probably rewritten anyways.

If you're adamant about removing them, I will though. Don't we have any other means of linking towards an API chapter where the link will seamlessly "migrate" along with the new API versions (let's say, as long as the "target section" name doesn't change, even though its absolute URL might) ?